src/main/kotlin/com/project/geo/dto/StreetResponse.kt

```diff
@@ -0,0 +1,10 @@
+package com.project.geo.dto
+
+import com.fasterxml.jackson.annotation.JsonCreator
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties
+import com.fasterxml.jackson.annotation.JsonProperty
+
+@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
why do we need it?

Reply:
I explained in comments from prev review. I added this on purpose. I don't need to de-serialize all properties